README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
The alternative version would be:

![redux-cycles-alt](https://cloud.githubusercontent.com/assets/315596/22519873/1bce6d96-e8c3-11e6-9a8f-aee46021b8d9.png)

and the one in this PR for reference:

![redux-cycles](https://cloud.githubusercontent.com/assets/315596/22519882/26d9c26c-e8c3-11e6-919c-daf721594ea4.png)